test/libsolidity/semanticTests/abiEncoderV2/abi_encode_v2_in_modifier_used_in_v1_contract.sol

```diff
@@ -39,4 +39,4 @@ contract C is B {
 // ----
 // test() -> 5, 10
 // gas irOptimized: 87337
-// gas legacy: 102637
+// gas legacy: 102651
```

Comment:
Hm, why did the gas change in all these files?

Reply:
Creation costs

Reply:
Well, something must have changed in evmone regarding it - wouldn't see it as a cause for worrying.